scripts/test-storj-sdk-aws.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+set -ueo pipefail
+
+#setup tmpdir for testfiles and cleanup
+TMPDIR=$(mktemp -d -t tmp.XXXXXXXXXX)
+cleanup(){
+	rm -rf "$TMPDIR"
+}
+trap cleanup EXIT
+
+SRC_DIR=$TMPDIR/source
+DST_DIR=$(mktemp -d -t tmp.XXXXXXXXXX)
+mkdir -p $SRC_DIR $DST_DIR
+
+aws configure set aws_access_key_id     $GATEWAY_0_ACCESS_KEY
+aws configure set aws_secret_access_key $GATEWAY_0_SECRET_KEY
+aws configure set default.region        us-east-1
+
+head -c 1024 </dev/urandom > $SRC_DIR/small-upload-testfile      # create 1mb file of random bytes (inline)
+head -c 5120 </dev/urandom > $SRC_DIR/big-upload-testfile        # create 5mb file of random bytes (remote)
```

Comment:
Wouldn't it make sense to have the `big-upload-testfile` as 65mb, so it is larger than one segment?